hayeon/programmers-1/짝수와-홀수.js

```diff
@@ -0,0 +1,15 @@
+function solution(num) {
+  if (num % 2 === 0) return "Even";
+  else {
+    return "Odd";
+  }
+}
```

Comment:
```suggestion
function solution(num) {
  if (num % 2 === 0) return "Even";
  return "Odd";
}
```
- 잘푸셨어요~ 밑에 else문은 불 필요한 블록입니다 :) 위에 if문에서 return이 있기 때문에 아래 else 블록은 없어도 될 것 같습니다 ! 
- 불 필요한 블록을 추가하는건 가독성 면에서 안좋은 것 같아요 😂